external/fv3fit/tests/training/_regtest_outputs/test_train.test_train_default_model_on_identity[sklearn_random_forest].out

```diff
@@ -1,3 +0,0 @@
-('var_in', '98a134e67c4a2b9a2da5d9744988191f')
```

Comment:
You probably already have, but make sure to delete all of these whose tests have been deleted.